author.rb

```diff
@@ -0,0 +1,17 @@
+require './item'
+
+class Author < Item
+  attr_accessor :id, :name, :items
+
+  def initialize(name)
+    super()
+    @id = rand(1...9999)
+    @name = name
+    @items = []
+  end
+
+  def add_item(_item)
+    @items << items
+    items.author = self
```

Comment:
when setting the author property of the item, you need to use the _item parameter, not the undefined variable items. So, it should be _item.author = self instead of items.author = self.